site/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["./src"]
+  "include": ["./src", "./scripts"],
+  "compilerOptions": {
+    "allowImportingTsExtensions": true
+  }
```

Comment:
`allowImportingTsExtensions` makes sense in the `site` because we use a bundler.